extensions/extensions-helm/aissemble-spark-operator-chart/values.template.yaml

```diff
@@ -48,7 +43,7 @@ spark-operator:
     - name: spark-logging
       mountPath: "/tmp/spark-events"
     - name: ivy-cache
-      mountPath: "/opt/spark/.ivy2"
+      mountPath: "/home/spark/.ivy2"
```

Comment:
I: The base Spark Operator image uses the default home location for the `spark` user. Technically we could have updated the home in the image build with `RUN usermod`, but this sets us up to remove the aissemble-spark-operator image entirely after the upgrade to Spark 3.5.